### DISCUSSION TITLE
How to adjust landscape PDF pages to match portrait width?

### DISCUSSION THREAD

**Discussion by holyboom** [stephanrauh/ngx-extended-pdf-viewer#1759] [Q&A]
How can I shrink landscape pages to have the same width as portrait for the PDF which has both orientations ?

[Expectation: **same result as thumbnails on the left**]
<img width="1014" alt="Screen Shot 2566-05-11 at 11 31 42" src="https://github.com/stephanrauh/ngx-extended-pdf-viewer/assets/6735861/ca079de8-7a79-4679-bd45-283bd1315b53">

Since Landscape mode sometimes affects mobile scrolling Y a-six when scrolling down.

Version info
- "ngx-extended-pdf-viewer": "^17.0.0-alpha.3",

To Reproduce

Use the PDF provided below:
Demo PDF file
[Portrait_Landscape.pdf](https://github.com/stephanrauh/ngx-extended-pdf-viewer/files/11449124/Portrait_Landscape.pdf)

**Comment by holyboom** [comment]
@stephanrauh please help 😔

**Reply by stephanrauh** [reply]
This requirement sounds like a new feature to me. I guess it makes sense, so I'm ready to support it. And - as you've pointed out yourself - the thumbnails view proves that it's possible.

If you want to implement the feature yourself: you'll probably have to adapt the file `pdf_page_view.js`. I suppose that's not enough: you also have to care about the annotation layers, and you have to fix scrolling and the `pdf_link_service.js`. So I don't think that the full-blown solution is that easy. However, a proof of concept should be in easy reach.

**Reply by holyboom** [reply]
Thank you so much for your clarification on this feature.Yes, it is not that easy especially the annotation layer part.

Really appreciate your support 🧡
